README.md

```diff
@@ -3,25 +3,28 @@
 
 ## Thank you for participating in our preview of config improvements
-As we prepare to launch a set of new configuration features we appreciate the opportunity to work with you in validating and improving on our design choices for the next major wave of CircleCI configuration improvements. If you were invited to the preview by your CircleCI account team, please arrange with them to get on our discussion forum. If you have stumbled upon this repository we welcome you to look around and give us feedback by submitting issues on this repo.
+As we prepare to launch a set of new configuration features we appreciate the opportunity to work with you in validating and improving on our design choices for the next major wave of CircleCI configuration improvements. If you were invited to the preview by your CircleCI account team, please arrange with them to get on our discussion forum. If you have stumbled upon this repository, we welcome you to look around and give us feedback by submitting issues on this repo.
 
 ## What problems are we trying to solve?
-1. **Better [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) support:** CircleCI 2.0 original configuration was highly deterministic but also highly verbose and forced too much boiler plate or repeated code blocks. We aim to provide better first-class semantics to dramatically improve the DRY aspects of CircleCI configuration.
+1. **Better [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) support** 
+CircleCI 2.0 original configuration was highly deterministic. It was also highly verbose and forced too many boiler plate and repeated code blocks. We aim to provide better first-class semantics to dramatically improve the DRY aspects of CircleCI configuration.
 
-2. **Code Reuse across projects:** We hear from a lot of customers that they want better ways to share configuration across projects. This repo introduces Orbs, reusable packages of parameterizable CircleCI configuration elements.
+2. **Code reuse across projects** 
+We hear from a lot of customers that they want better ways to share configuration across projects. This repo introduces Orbs, reusable packages of parameterizable CircleCI configuration elements.
 
-3. **Easier path to common configuration:** We hear often that people want more off-the-shelf options to get their first useful builds flowing, especially for common platforms like Rails, Node, and they want better encapsulation of common tasks like deploying to Heroku or pushing to an S3 bucket.
+3. **Easier path to common configuration**
+We often hear that people want more off-the-shelf options to get their first useful builds flowing, especially for common platforms like Rails, Node, and they want better encapsulation of common tasks like deploying to Heroku or pushing to an S3 bucket.
 
-## Major Features Previewed Here
+## Major features previewed here
```

Comment:
Should this list include things like pre- and post- steps, or link to the respective docs?

Reply:
Links would be good, but I am not holding up getting this merged to do it.